doc/pyvista-binder-config.yml

```
@@ -3,4 +3,4 @@ default_context:
     github_account: "pyvista"
     description: "Quick examples demonstrating what PyVista can do! Simply launch this Binder project and test drive PyVista"
     author_name: "PyVista Developers"
-    year: "2022"
+    year: "2022-2023"
```

```suggestion
    year: "2023"
```

I don't think this needs to be a range and the active year is sufficient